### PR TITLE
Add com.synology.SynologyAssistant

### PR DESCRIPTION
**Pull request by hadess**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.synology.SynologyAssistant.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+app-id: com.synology.SynologyAssistant
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '20.08'
```
Comment (hfiguiere) [inline com.synology.SynologyAssistant.yaml:4]: not 21.08?

File: com.synology.SynologyAssistant.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+sdk: org.freedesktop.Sdk
+command: synology-assistant
+tags: [proprietary]
+finish-args:
+  - --share=network
+  - --socket=x11
```
Comment (hfiguiere) [inline com.synology.SynologyAssistant.yaml:10]: I believe `--share=ipc` is still needed for anything X11.

**Comment by hfiguiere** [comment]
bot, build com.synology.SynologyAssistant

File: com.synology.SynologyAssistant.yaml (hunk 2 above)
Comment (hadess) [reply com.synology.SynologyAssistant.yaml:10]: That wasn't needed for X11 on Wayland, but that sounds correct.

**Comment by hadess** [comment]
Thanks for the quick reviews!

**Pull request closed without merging**